lib/plugins/chat.js

```diff
@@ -1,8 +1,10 @@
-const CHAT_LENGTH_LIMIT = 100
+const PROTO_VER_1_10 = require('minecraft-data')('1.10.2').version.version
```

Comment:
why not hardcode this to `301`?

Reply:
Among other reasons, because it's ugly A.F., and confusing to newcomers and potential maintainers. The more explicit we are, the better.